webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/page/ApplicationPageBase.html

```diff
@@ -47,7 +47,7 @@
     </wicket:child>
   </div>
 
-  <div wicket:id="feedbackPanel" class="feedback-panel"></div>
+  <div wicket:id="feedbackPanel" class="feedback-panel" style="z-index: 9999"></div>
```

Comment:
How about moving the z-level into the `.feedback-panel` class in `custom.less`?

Reply:
Makes sense, will do.